Homework/Develop/assets/css/style.css

```diff
@@ -8,39 +8,43 @@ body {
     background-color: #d9dcd6;
 }
 
-.header {
+
+
+header {
     padding: 20px;
     font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
     background-color: #2a607c;
     color: #ffffff;
 }
 
-.header h1 {
+header h1 {
     display: inline-block;
     font-size: 48px;
 }
 
-.header h1 .seo {
+header h1 .seo {
     color: #d9dcd6;
 }
 
-.header div {
+Nav {
     padding-top: 15px;
     margin-right: 20px;
     float: right;
     font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
     font-size: 20px;
 }
 
```

Comment:
Maybe worth adding some comments for sections of your style sheet in improve readability.

**NOTE:** `/*  This is a comment in CSS */`

Reply:
added the comments within the CSS file here dd90339a4aa6736e8ec16f222bd65186e4b010a9